docs/reference/modules/cross-cluster-search.asciidoc

```diff
@@ -372,3 +372,16 @@ image:images/ccs/ccs-dont-min-roundtrip-shard-results.svg[]
 final results in the {ccs} response.
 +
 image:images/ccs/ccs-min-roundtrip-client-response.svg[]
+
+[float]
+[[ccs-supported-configurations]]
+==== Supported configurations
```

Comment:
```suggestion
[discrete]
[[ccs-supported-configurations]]
=== Supported configurations
```

Reply:
Suggest changing the heading to level 3 `===` so that the section displays in the right-hand navigation. Otherwise, users have to scroll all the way to the bottom of this page.